paddle/utils/Flags.cpp

```diff
@@ -20,6 +20,14 @@ DEFINE_bool(use_gpu, false, "Only support CPU training");
 DEFINE_bool(use_gpu, true, "Whether to use GPU for training");
 #endif
 
+#ifdef PADDLE_USE_MKLDNN
+// TODO(TJ): change to true when MKLDNN layers support multi-inputs
+DEFINE_bool(use_mkldnn, false, "Default still keep use CPU training");
```

Comment:
I don't get it --  why would we need these flags. We had a discussion that we shouldn't add new flags, because they are actually global variables.  For this specific PR, are these flags necessary?